types/react-interactive/index.d.ts

```diff
@@ -0,0 +1,85 @@
+// Type definitions for react-interactive 0.9
+// Project: https://github.com/rafrex/react-interactive
+// Definitions by: Daniel <https://github.com/DanudeSandstorm>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+import { ReactNode, ReactElement, Component, SyntheticEvent, CSSProperties } from 'react';
+
+export {};
+
+export type iState = 'normal' | 'hover' | 'hoverActive' | 'touchActive' | 'keyActive';
+export type focus = false | 'tab' | 'mouse' | 'touch';
+export type clickType = "mouseClick" | "tapClick" | "keyClick";
```

Comment:
Type names should typically be PascalCase (even unions of unit types) unless there’s a really good reason to do otherwise.